### PR TITLE
add registration verification form template

### PR DESCRIPTION
**Pull request by jesopo**
we may wish to, perhaps in the future, not have this page use a layout that excludes the navbar, but at the moment the navbar includes relative URLs, and because this template will be hosted somewhere else, those won't work

### CODE REVIEW

**Comment by jesopo** [comment]
this solution isn't sufficient. the jekyll-compiled output for this page includes relative URLs, which won't work when that output is taken to a different domain/path to be rendered by curite

File: registration/form.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: Libera Chat account verification
+layout: nohead
+---
+
+{% raw %}
```
Comment (swantzter) [inline registration/form.md:6]: since you use raw and nomarkdown so much here, I'd just make this a html file

**Comment by swantzter** [comment]
> this solution isn't sufficient. the jekyll-compiled output for this page includes relative URLs, which won't work when that output is taken to a different domain/path to be rendered by curite

What links does it include now? Stuff in the <head/>?

**Comment by jesopo** [comment]
blocked by #255

File: registration/verify.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+{% raw %}
+{::nomarkdown}
+<!-- markdownlint-disable MD033 -->
+
+<div class="verification">
+  <form method="POST" action="{{target}}">
+    <input type="submit" value="Verify">
+  </form>
+  <script>
+    /*
```
Comment (swantzter) [inline registration/verify.md:15]: You probably want to uncomment this js before merging?

Comment (jesopo) [reply registration/verify.md:15]: yes, you're right, but also is that js going to work any more? it feels like i lost the ID it's referring to

Comment (swantzter) [reply registration/verify.md:15]: Ah, yeah. You need to add that back

File: registration/verify.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+title: Libera Chat account verification
+absolutify: true
+---
+
+{% raw %}
+{::nomarkdown}
+<!-- markdownlint-disable MD033 -->
+
+<div class="verification">
+  <form method="POST" action="{{target}}">
```
Comment (swantzter) [inline registration/verify.md:11]:
```suggestion
  <form method="POST" action="{{target}}" id="verification-form">
```

**Pull request merged**